### DISCUSSION TITLE
Strange issue with installation.

### DISCUSSION THREAD

**Discussion by leet0rz** [neovim/neovim#25616] [General]
### Problem

Ran the installer with:
```
./nvim.appimage --appimage-extract
./squashfs-root/usr/bin/nvim
```

but when I try to run it I get `command not found: nvim`

Where exactly does the nvim file go?

### Steps to reproduce

can't run this either.

### Expected behavior

For neovim to install and launch with nvim.

### Neovim version (nvim -v)

NVIM v0.10.0

### Vim (not Nvim) behaves the same?

vim runs

### Operating system/version

WSL

### Terminal name/version

zsh

### $TERM environment variable

xterm-256color

### Installation

app.image from release, latest.

**Comment by justinmk** [comment]
`--appimage-extract` merely "unzips" the files, it doesn't install nvim anywhere.

You can run nvim without installing by running `./squashfs-root/usr/bin/nvim` (or look for `nvim` extracted files and then figure out the path from that.

**Comment by leet0rz** [comment]
> `--appimage-extract` merely "unzips" the files, it doesn't install nvim anywhere.
> 
> You can run nvim without installing by running `./squashfs-root/usr/bin/nvim` (or look for `nvim` extracted files and then figure out the path from that.

nvim opens after I run the original 2 lines, but when I close it again I can not run `./squashfs-root/usr/bin/nvim` or `nvim`, I cannot locate the nvim file either to run it.


`no such file or directory: ./squashfs-root/usr/bin/nvim` when I try to run what you gave me.

Any ideas?

**Comment by justinmk** [comment]
> `no such file or directory: ./squashfs-root/usr/bin/nvim` when I try to run what you gave me.

Sounds like your directory changed. You need to keep track of your files, I don't know how to help you.

**Comment by leet0rz** [comment]
> > `no such file or directory: ./squashfs-root/usr/bin/nvim` when I try to run what you gave me.
> 
> Sounds like your directory changed. You need to keep track of your files, I don't know how to help you.

Ah. I ran it from my c drive, I thought it would target the root of linux for some reason but I guess it just creates squashfs-root folder exactly where it is. How exactly would I make it work with my WSL? Now I found the folder and it sits in `~` is it possible to run it so that it would be in my WSL PATH/Linux folders and I can just `nvim` in the WSL terminal? Thanks :)

**Comment by leet0rz** [comment]
For anyone else encountering this issue. I just moved all the files inside the squashfs-root into /, basically the base of the system. So all the files in /squashfs-root/usr into /usr etc for every folder from /squashfs-root which was extracted from appimage.

**Reply by SamGoodSpeed** [reply]
I had same issue = Thanks for helping!
